src/gbt_model_builder.pyx

```diff
@@ -179,3 +181,110 @@ def gbt_reg_model_builder(n_features, n_iterations):
     :param size_t n_iterations: Number of trees in the model
     '''
     return gbt_regression_model_builder(n_features, n_iterations)
+
+
+class GBTDAALModel:
```

Comment:
It's not obvious why this class should be in compiled file, not usual python script.

Reply:
> It's not obvious why this class should be in compiled file, not usual python script.

What benefits do you expect from moving this class from a compile file?